e2e/toasts.e2e.ts

```diff
@@ -35,3 +39,66 @@ test("Toast warn", async ({ page }) =>
 
 test("Toast message", async ({ page }) =>
   testToast({ page, toast: "toast-message" }));
+
+test("Should close toast", async ({ page }) => {
```

Comment:
On one side, lazyness. It was a really long, long day and I was like "f it" it's faster to implement screenshot comparison.

On the other side though, since I moved in this PR some tests because I faced issue with animations, ensuring the screenshots are correct also check that the UI is correct. Does not check that the animation was smooth but a least performed.

Works for you?

Reply:
I like this approach too. Let's try it and see how maintainable it is. Thanks for the explanation.

Reply:
Thanks!